src/psyclone/tests/test_files/dynamo0p3/infrastructure/mpi_mod.f90

```diff
@@ -63,10 +63,13 @@ module mpi_mod
          get_mpi_datatype, &
          get_comm_size, get_comm_rank
 
+  ! JH: Define the values for a single process. This way this
+  ! module can be used without special initialisation, which
+  ! means the LFRic ! profiling library can be used in GOcean.
```

Comment:
s/LFRic !/LFRic/

Reply:
Please update the copyright date for the modifications too.

Reply:
Done.